src/components/ConsentPicker.vue

```diff
@@ -0,0 +1,486 @@
+                <label for="consent-time" class="text-sm w-full">Expected Time Amount</label>
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div v-if="knowWhatIWant" class="flex flex-column pl-3" style="width: 50%">
+        <h3>Select a Consent Form</h3>
+        <PvDropdown
+          v-model="selectedConsent"
+          :options="listOfDocs.consent"
+          optionLabel="fileName"
+          style="width: 70%"
+          @change="updateConsent"
+        />
+        <h3 class="pt-3">Select an Assent Form</h3>
+        <PvDropdown
+          v-model="selectedAssent"
+          :options="listOfDocs.assent"
+          optionLabel="fileName"
+          style="width: 70%"
+          @change="updateAssent"
+        />
+        <div div class="hidden">
+          <h3 class="mb-4 mt-5">Consent Amount and Expected Time</h3>
+          <div class="flex flex-row">
+            <div class="mr-1">
+              <span class="p-float-label">
+                <PvInputText id="consent-amount" v-model="amount" class="w-full" disabled="true" />
+                <label for="consent-amount" class="text-sm w-full">Payment Amount $$</label>
+              </span>
+            </div>
+            <div class="ml-3">
+              <span class="p-float-label">
+                <PvInputText id="consent-time" v-model="expectedTime" class="w-full" disabled="true" />
+                <label for="consent-time" class="text-sm w-full">Expected Time Amount</label>
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div v-if="knowWhatIWant || helpMeSelect" class="flex-column" style="width: 50%">
+        <h3 class="font-bold text-center text-xl">Suggested Forms</h3>
+        <div class="w-full">
+          <PvFieldset legend="Consent" v-if="consents && consents.length > 0">
+            <div class="flex flex-row w-full">
+              <div style="width: 80%">
+                <p class="m-0">
+                  <span class="font-bold">Name: </span>{{ result.consent[0]?.fileName }} <br />
+                  <span class="font-bold">Current Commit: </span>{{ result.consent[0]?.currentCommit }}
+                  <br />
+                  <span class="font-bold">GitHub Org: </span>{{ result.consent[0]?.gitHubOrg }} <br />
+                  <span class="font-bold">GitHub Repository: </span>{{ result.consent[0]?.gitHubRepository }}
+                  <br />
+                  <span class="font-bold">Last Updated: </span>{{ result.consent[0]?.lastUpdated }} <br />
+                </p>
+              </div>
+              <div class="flex align-items-center justify-content-center">
+                <PvButton
+                  class="border-circle w-6rem h-6rem m-2 surface-hover text-primary border-none font-bold flex align-items-center justify-content-center hover:text-100 hover:bg-primary"
```

Comment:
Could we pass in a prop to `disable` this button when no form has been selected?